vizro-core/examples/default/app.py

```diff
@@ -526,7 +526,19 @@ def create_home_page():
         pages={
             "Analysis": ["Homepage", "Variable Analysis", "Relationship Analysis", "Country Analysis"],
             "Summary": ["Continent Summary"],
-        }
+        },
+        nav_selector=vm.NavBar(
+            items=[
+                vm.NavLink(
+                    label="Label 1",
+                    pages={"Analysis": ["Variable Analysis", "Homepage"], "Summary": ["Continent Summary"]},
+                ),
```

Comment:
Can we keep the same attributes order like below? So:
```
                vm.NavLink(
                    pages={"Analysis": ["Variable Analysis", "Homepage"], "Summary": ["Continent Summary"]},
                    label="Label 1",
                ),
```

Reply:
I think these changes were probably just to test it out rather than meant to remain as changes to the default app.py?